index.html

```diff
@@ -289,6 +289,25 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact-me">
+        <header class="contact-me-header">
+          <h1>
+            I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h1>
+        </header>
+        <form onsubmit="validate()" action="https://formspree.io/f/mbjqyrqj" method="POST">
+          <fieldset tabindex="0">
+            <input type="text" name="name"
+              id="fullname" maxlength="30" placeholder="Full name">
+            <!-- <input type="email" required name="_replyto" id="email"> -->
```

Comment:
- [ ] REQUIRED: Remove all commented code. It a bad code practice and makes your code clean.